recipes/ckan-deploy.rb

```diff
@@ -33,6 +35,8 @@
 ckan_cli = "#{virtualenv_dir}/bin/ckan_cli"
 install_dir = "#{virtualenv_dir}/src/#{service_name}"
 
+log "#{DateTime.now}: Creating files and directories for CKAN"
```

Comment:
Can you prefix the logs with something to find them easier, since date always changes.
i.e. the script name we are running?
ckan-deploy: #{DATETIME.now}: ${message}

Reply:
Not really needed IMO, since the log entries will be surrounded by task logs that will tell us exactly where we are. This is just to get timestamps.